python/katana/benchmarking/bench_python_cpp_algos.py

```diff
@@ -17,11 +17,17 @@
 # TODO(giorgi): This script needs to be tested in CI.
 PathExt = namedtuple("PathExt", ["warn_prefix", "path_ext"])
```

Comment:
I'm not sold of having this in `katana.benchmarking` and hence installed. This is a quite special purpose and hard to use script. I think I would rather have this in `python/tests/benchmarking`. Is that reasonable? It would prevent it from being installed, which I think is good.

Reply:
@gliga Let me know if this change would still work for you?

Reply:
Yes, I like the suggested change.